students/cocokaku/lesson2/fizzbuzz.py

```diff
@@ -0,0 +1,5 @@
+for i in range(1, 101):
+    if i%15==0: print("FizzBuzz")
+    elif i%3==0: print("Fizz")
+    elif i%5==0: print("Buzz")
+    else: print(i)
```

Comment:
although it's a valid syntax, this is against python style guidelines; make sure to use separate lines for if blocks:
```
if i % 15 == 0:
    print("FizzBuzz")
```